deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -259,6 +259,7 @@ async def create_new_job(
         logger.info(f"Creating pulse3d job for upload {details.upload_id} with user ID: {user_id}")
 
         params = [
+            "normalize_y_axis",
```

Comment:
Depending on which version of Pulse3D this param gets added, you'll need to conditionally add it in. See lines 271-281